integration/send_runlog_transaction.js

```diff
@@ -0,0 +1,60 @@
+// truffle script
```

Comment:
Can we move all these loose js scripts into `integration/scripts`? This directory is already a truffle project, and that would make it organized a bit more like one, making it more intuitive the next dev.